plugin/evm/message/cross_chain_handler.go

```diff
@@ -43,19 +43,19 @@ func (c *crossChainHandler) HandleEthCallRequest(ctx context.Context, requesting
 	transactionArgs := ethapi.TransactionArgs{}
 	err := json.Unmarshal(ethCallRequest.RequestArgs, &transactionArgs)
 	if err != nil {
-		log.Debug("error occurred with JSON unmarshalling ethCallRequest.RequestArgs", "err", err)
+		log.Error("error occurred with JSON unmarshalling ethCallRequest.RequestArgs", "err", err)
 		return nil, nil
 	}
 
 	result, err := ethapi.DoCall(ctx, c.backend, transactionArgs, lastAcceptedBlockNumberOrHash, nil, c.backend.RPCEVMTimeout(), c.backend.RPCGasCap())
 	if err != nil {
-		log.Debug("error occurred with EthCall", "err", err, "transactionArgs", ethCallRequest.RequestArgs, "blockNumberOrHash", lastAcceptedBlockNumberOrHash)
+		log.Error("error occurred with EthCall", "err", err, "transactionArgs", ethCallRequest.RequestArgs, "blockNumberOrHash", lastAcceptedBlockNumberOrHash)
 		return nil, nil
 	}
 
 	executionResult, err := json.Marshal(&result)
 	if err != nil {
-		log.Debug("error occurred with JSON marshalling result", "err", err)
+		log.Error("error occurred with JSON marshalling result", "err", err)
```

Comment:
These requests should only come from a VM running on the same node, so failing to serve the request should be considered a warning at this time.

Note: `CrossChainHandler` is not used by any VM at this time

Reply:
Should these be a `Warn`? not an `Error`?

Reply:
This is assuming that requests coming from the same node. This is a feature that's not in use atm, so it depends on if VMs that wanted to use this should only send requests that should be successful 🤔

That was my logic when updating to `Error` at least.

Reply:
As of right now, if this log showed up I'd want to know about it rather than only wanting to know about it if it showed up in a large quantity or a surge, so I'm going to leave this as is for now.

We may want to change this in the future if the access pattern for it changes.